### DISCUSSION TITLE
The memes...

### DISCUSSION THREAD

**Discussion by ashwinvk94** [facontidavide/PlotJuggler#474] [General]
This is just an appreciation post for the great memes that make my day when I launch PlotJuggler. Who makes them?

**Comment by facontidavide** [comment]
me, using https://imgflip.com/, but please feel free to submit your own!

**Reply by facontidavide** [reply]
![image](https://user-images.githubusercontent.com/2822888/124153881-38089180-da95-11eb-91f3-b70fe7708e65.png)

**Reply by ashwinvk94** [reply]
lol. Will do if I think of anything. Thanks for writing this amazing piece of code.
